types/lodash/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es2023"
```

Comment:
Does this mean lodash users can no longer target ES <2023?

Reply:
i don't think so, but i have no idea how to get the `WeakMap` type to be compatible on both targets

Reply:
It might be time to remove this declaration. The comment says it's for es5 compatibility.